src/messages/messagesNN.ts

```diff
@@ -97,6 +97,52 @@ const messages = {
       userUpdated: "Bruker oppdatert",
     },
     goToMyNdla: "Gå til Min NDLA",
+    learningpath: {
+      title: "Mine læringsstiar",
+      description:
+        "Her kan du lage dine eigne læringsstiar og dele dei med elevane dine. Læringsstiane kan innehalde artiklar frå NDLA, lenkjer til andre ressursar samt korte tekstar du lagar sjølv. Sjå døme på ein læringssti www.lenketildøme.no",
```

Comment:
Vi har brukt lenke og ikkje lenkje ellers i språkfilene så bruk det er også. Laga ikkje Dave ein pr i frontend-packages med oversettelser? Kanskje legge til tekstene der også?

Reply:
Jeg har egentlig bare skrevet de på nytt, og ser vi har egentlig akkurat de samme tekstene. Han har bare brukt litt fra mapper som jeg gjorde! Vi tenker å ha disse i NDLA-frontend repoet framfor frontend-packages, så ernte no vits å legge de til i FP

Reply:
Vi har masse tekster i frontend-packages som kun brukes i ndla-frontend. Synes vi burde være konsekvente her. Enten ha alt i frontend-packages, eller begynne å fjerne ting som kun finnes i ndla-frontend.

Reply:
Stemmer for sistnevnte